MovieApp/MovieApp/MovieCell.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+class MovieCell: UICollectionViewCell {
+    
+    static let cellIdentifier = "cellid"
+    
+    let titleFontSize: CGFloat = 16
+    let descriptionFontSize: CGFloat = 14
+    let offset: CGFloat = 8
+    let movieImageWidth: CGFloat = 97
+    let movieTitleViewHeight: CGFloat = 20
+    
+    var movieTitle: UILabel!
+    var movieDescription: UILabel!
+    var movieImageView: UIImageView!
+    var cellView = UIView()
```

Comment:
Since you've created all views in `createViews` function, please do it for the `cellView` as well.
Also, you could name it a bit better (`contentContainer` or sth like that)